editor/components/default-block-appender/index.js

```diff
@@ -24,7 +25,7 @@ export function DefaultBlockAppender( { isLocked, isVisible, onAppend, showPromp
 		return null;
 	}
 
-	const value = placeholder || __( 'Write your story' );
+	const value = decodeEntities( placeholder ) || __( 'Write your story' );
```

Comment:
Should we decode entities in the translated string as well, in case the translation includes entities?

Might speak to a broader point of whether we should want to bake this into the behavior of `__` automatically.

Reply:
Baking it into `__` automatically sounds like a good idea, I wonder if there's a case where you wouldn't want this in a translation.